### PR TITLE
chore: docs and templates update

### PR DESCRIPTION
**Pull request by BrendaHijmans**
closes #https://github.com/molgenis/GCC/issues/430

NOTE: links to images and xlsx will not work, because they point to master code. See the files changed for the updated templates and images

What are the main changes you did:
- docs and templates for metadata upload changed

how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Review by EleanorHyde-UMCG — APPROVED** [review]

File: docs/catalogue/cat_cohort-data-manager.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+[MOLGENIS catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
+framework to describe in detail: metadata of different data resources, such as cohorts and data sources; definitions of 
+the variables collected in these resources; and mappings to common data models. Its purpose is to provide a gateway to 
+find and access health research data and to facilitate pooled data analysis of multiple cohorts 
+[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data source studies 
+[Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
 
-- The metadata of cohorts include descriptive information such as contact details, name of the cohort, and high-level
-  summary of contents and cohort design. The metadata of data sources, of the corresponding data banks and of the
+- The metadata of cohorts include descriptive information such as contact details, name of the resource, and a high-level
```
Comment (hslh) [inline docs/catalogue/cat_cohort-data-manager.md:12]: 'The metadata of cohorts' -> 'The metadata of resources'/'The metadata of data resources'

File: docs/catalogue/cat_network-data-manager.md [hunk 2]
```diff
@@ -1,58 +1,56 @@
...
-to facilitate pooled data analysis of multiple cohorts.
+[MOLGENIS catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
+framework to describe in detail: metadata of different data resources, such as cohorts and data sources; definitions of 
+the variables collected in these resources; and mappings to common data models. Its purpose is to provide a gateway to 
+find and access health research data and to facilitate pooled data analysis of multiple cohorts 
+[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data source studies 
+[Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
 
-- The cohort metadata provides descriptive information such as contact details, name of the cohort, and high-level
+- The resource metadata provides descriptive information such as contact details, name of the cohort, and high-level
```
Comment (hslh) [inline docs/catalogue/cat_network-data-manager.md:16]: 'name of the cohort' -> 'name of the resource'

File: docs/catalogue/cat_network-data-manager.md [hunk 3]
```diff
@@ -1,58 +1,56 @@
...
+## MOLGENIS catalogue
 
 ### Define CDM metadata
 
-[MOLGENIS Data Catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#/variable-explorer/) (sometimes
-also called 'EMX2 catalogue') provides a framework to describe in detail: cohort metadata; definitions of the data
-variables collected (aka 'source variables'); and mappings to common data models (aka 'target variables). Its purpose is
-to facilitate pooled data analysis of multiple cohorts.
+[MOLGENIS catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
+framework to describe in detail: metadata of different data resources, such as cohorts and data sources; definitions of
```
Comment (timcadman) [inline docs/catalogue/cat_network-data-manager.md:10]: I would probably say "... data resources (such as cohorts and data sources); ..."

File: docs/catalogue/cat_network-data-manager.md [hunk 4]
```diff
@@ -1,58 +1,56 @@
...
 
-[MOLGENIS Data Catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#/variable-explorer/) (sometimes
-also called 'EMX2 catalogue') provides a framework to describe in detail: cohort metadata; definitions of the data
-variables collected (aka 'source variables'); and mappings to common data models (aka 'target variables). Its purpose is
-to facilitate pooled data analysis of multiple cohorts.
+[MOLGENIS catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
+framework to describe in detail: metadata of different data resources, such as cohorts and data sources; definitions of 
+the variables collected in these resources; and mappings to common data models. Its purpose is to provide a gateway to 
+find and access health research data and to facilitate pooled data analysis of multiple cohorts 
+[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data source studies
```
Comment (timcadman) [inline docs/catalogue/cat_network-data-manager.md:13]: datasource?

File: docs/catalogue/cat_network-data-manager.md [hunk 5]
```diff
@@ -1,58 +1,56 @@
...
   ALSPAC).
 - Similarly, the common data model metadata (or 'target variables') can be considered the codebook for a network of
   cohorts working together (e.g. LifeCycle)
 - The mappings describe how source variables have been converted into target variables as basis for integrated analysis.
 
 This section explains how to submit the 'target variables' (also called the harmonised model or common data model) into
-the Data Catalogue. Expected users of this 'how to' are central data managers of networks such as LifeCycle or
-LongITools. You will need login details to upload metadata to MOLGENIS Data Catalogue.
+the catalogue. Expected users of this 'how to' are central data managers of networks such as LifeCycle or
+LongITools. You will need login details to upload metadata to MOLGENIS catalogue.
```
Comment (timcadman) [inline docs/catalogue/cat_network-data-manager.md:26]: "the MOLGENIS catalogue"

File: docs/catalogue/cat_network-data-manager.md [hunk 6]
```diff
@@ -1,58 +1,56 @@
...
 
 #### Define common data elements
 
 We use the [*TargetDictionary* template](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/TargetDictionary.xlsx)
-to describe the common data model elements. The 
+to describe common data model elements. The 
 [*TargetDictionary* template](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/TargetDictionary.xlsx)
-consists of multiple sheets. Each sheet corresponds to a table in the Data Catalogue. The columns in the sheet
+consists of multiple sheets. Each sheet corresponds to a table in catalogue. The columns in the sheet
```
Comment (timcadman) [inline docs/catalogue/cat_network-data-manager.md:33]: "the catalogue"

File: docs/catalogue/cat_network-data-manager.md [hunk 7]
```diff
@@ -1,58 +1,56 @@
...
 accept a data upload. You can download this
 [*filled out example*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/TargetDictionary_testCDM.xlsx)
-as a reference for filling out the template. Note that there is no sheet for *All variables*. This table is a generic listing of all
-variables entered for the cohort; it shows *Variables* and *Repeated variables* in one table.
+as a reference for filling out the template.
 
 It is good practice to try adding a few variables to the template first and see whether your upload succeeds. To
-upload the metadata to the Data Catalogue see the section To upload the metadata to the Data Catalogue see the
-section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Data Catalogue.
+upload the metadata to the catalogue see the section To upload the metadata to the catalogue see the
```
Comment (timcadman) [inline docs/catalogue/cat_network-data-manager.md:41]: Something has gone wrong with this sentence!

**Review by timcadman — CHANGES_REQUESTED** [review]
Just made a few suggestions on the English but otherwise looks good :)

File: docs/catalogue/cat_network-data-manager.md (hunk 5 above)
Comment (timcadman) [reply docs/catalogue/cat_network-data-manager.md:26]: Should read 'the' MOLGENIS catalogue

**Review by timcadman — APPROVED** [review]
Just one more 'the' that needs to be added but happy to approve it

**Pull request merged**